dotnet/xml/ns-Microsoft.Identity.Client.Broker.xml

```diff
@@ -1,6 +1,72 @@
 <Namespace Name="Microsoft.Identity.Client.Broker">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary overwrite="false">
+        <format type="text/markdown"><![CDATA[
+[![NuGet](https://img.shields.io/nuget/v/Microsoft.Identity.Client.Broker.svg?style=flat-square&label=Microsoft.Identity.Client.Broker&colorB=00b200)](https://www.nuget.org/packages/Microsoft.Identity.Client.Broker/)  [![NuGet downloads](https://img.shields.io/nuget/dt/Microsoft.Identity.Client.Broker?style=flat-square)](https://www.nuget.org/packages/Microsoft.Identity.Client.Broker/) [![GitHub Repo stars](https://img.shields.io/github/stars/AzureAD/microsoft-authentication-library-for-dotnet?style=flat-square)](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet)
+
+The `Microsoft.Identity.Client.Broker` is the namespace in the Microsoft Authentication Library (MSAL) for .NET that handles interactions with authentication brokers. It is available through the [`Microsoft.Identity.Client.Broker`](https://www.nuget.org/packages/Microsoft.Identity.Client.Broker/) package.
+
+This package requires the use of [`Microsoft.Identity.Client`](xref:Microsoft.Identity.Client).
+
+>[!NOTE]
+>The `Microsoft.Identity.Client.Broker` functionality is not available for UWP or legacy applications using .NET Framework or .NET Core 3 or below.
```

Comment:
or apps targeting net6-windows.
Maybe we need to link to the doc that shows the applicability of Msal.Broker?

Reply:
Updated the verbiage - think this could be a good start, and we expand the WAM document.